x-pack/plugins/apm/public/components/app/service_overview/latency_chart/index.tsx

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiFlexGroup, EuiFlexItem, EuiSelect, EuiTitle } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import React, { useState } from 'react';
+import { FETCH_STATUS } from '../../../../hooks/useFetcher';
+import { useTransactionCharts } from '../../../../hooks/useTransactionCharts';
+import { TimeseriesChart } from '../../../shared/charts/timeseries_chart';
+import { getResponseTimeTickFormatter } from '../../../shared/charts/transaction_charts/helper';
+import { useFormatter } from '../../../shared/charts/transaction_charts/use_formatter';
+
+interface Props {
+  height?: number;
+}
+
+export function LatencyChart({ height }: Props) {
+  const [selectedOption, setSelectedOption] = useState(0);
+  const { data, status } = useTransactionCharts();
+  const { responseTimeSeries, anomalySeries } = data;
+  const timeSeries = responseTimeSeries
+    ? [responseTimeSeries[selectedOption]]
+    : [];
+  const { formatter } = useFormatter(timeSeries);
```

Comment:
Tangential: I dived into `useFormatter` and found the use of `getMaxY` helper. I think it's something you added earlier and I just noticed some small improvements:

```ts
export function getMaxY(timeSeries: Array<TimeSeries<Coordinate>>) {
  const coordinates = timeSeries.flatMap((serie) => serie.data);
  const yValues = coordinates.map((c) => c.y ?? 0);
  return Math.max(...yValues, 0);
}
```

Notice how we all the manual types were dropped? This is both cleaner and more robust, since we use the type system.

It will require you to change `TimeSeries` to `TimeSeries<Coordinate>` in a couple of places but I think it's worth it since we get better types this way